client/lib/plans/constants.js

```diff
@@ -378,11 +384,13 @@ export const PLANS_LIST = {
 			FEATURE_MALWARE_SCANNING_DAILY,
 		] ),
 		getSignupFeatures: () => compact( [
-			FEATURE_WORDADS_INSTANT,
 			FEATURE_MALWARE_SCANNING_DAILY,
+			FEATURE_VIDEO_CDN_LIMITIED,
+			FEATURE_WORDADS_INSTANT,
 			FEATURE_ALL_PERSONAL_FEATURES,
 		] ),
 		getBillingTimeFrame: () => i18n.translate( 'per year' ),
+		getSignupBillingTimeFrame: () => 'per year', //PLANS A/B TEST: Translate if test passes
```

Comment:
Any good reason to not wrap this one in a `translate` call immediately? This string is already translated in the above line.